adapters/missena/missena.go

```diff
@@ -124,26 +209,20 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
 			continue
 		}
 
-		var missenaExt openrtb_ext.ExtImpMissena
+		var missenaExt *openrtb_ext.ExtImpMissena
 		if err := jsonutil.Unmarshal(bidderExt.Bidder, &missenaExt); err != nil {
 			errors = append(errors, &errortypes.BadInput{
 				Message: "Error parsing missenaExt parameters",
 			})
 			continue
 		}
 
-		missenaInternalParams.ApiKey = missenaExt.ApiKey
-		missenaInternalParams.Placement = missenaExt.Placement
-		missenaInternalParams.TestMode = missenaExt.TestMode
-
-		newHttpRequest, err := a.makeRequest(missenaInternalParams, requestInfo, imp.ID, request)
+		newHttpRequest, err := a.makeRequest(imp, request, requestInfo, missenaExt, gdprApplies, consentString)
 		if err != nil {
 			errors = append(errors, err)
 			continue
 		}
-
 		httpRequests = append(httpRequests, newHttpRequest)
-
 		break
```

Comment:
break in the loop causes only the first imp to be processed. If this is intentional, it's worth making it clearer in the code, e.g., with a comment or better naming.